NiBAx/plugins/SmileGAN/SmileGAN.py

```diff
@@ -0,0 +1,292 @@
+from PyQt5.QtGui import *
+from matplotlib.backends.backend_qt5 import FigureCanvasQT
+from PyQt5 import QtGui, QtCore, QtWidgets, uic
+import joblib
+import sys, os, time
+import seaborn as sns
+import numpy as np
+import pandas as pd
+from NiBAx.core.plotcanvas import PlotCanvas
+from NiBAx.core.baseplugin import BasePlugin
+from NiBAx.core.gui.SearchableQComboBox import SearchableQComboBox
+
+class computeSPAREs(QtWidgets.QWidget,BasePlugin):
```

Comment:
Without going through every single line, this class seems to be a copy of computespares.py. If so, we might not need to duplicate it here, we could just instantiate that one.